static/js/index.js

```diff
@@ -1,3 +1,8 @@
+const FALL_TRI_START = new Date(2022, 9 - 1, 6, 15, 30, 0, 0)
```

Comment:
There are variables sent down in index.html that mark the end of the trimesters:
```
    var fallTriEndDate = new Date({{fall_end_unix | safe}});
    var wintTriEndDate = new Date({{wint_end_unix | safe}});
```
Let's try to use those variables rather than hardcoding dates here in the JS.

In fact, you might be able to just call the existing `skipToWinter` and `skipToSpring` functions.

Reply:
I feel like the array setup works better, plus there isn't a skipToFall one

Reply:
sure, the array is fine. My main concern is that this stuff shouldn't be hard-coded in the JS (we always have problems at the start of the school year due to this sort of thing) so if you need the fall start date, let's send it down from the server. We could send an array instead of 3 variables to make it easy for you.

Reply:
I started working on an update that automatically reads the trimester start dates, so I can take care of most of this work for you.

Reply:
See #546